adoc/deployment-cilium.adoc

```diff
@@ -40,3 +45,12 @@ https://cilium.readthedocs.io/en/v1.5/gettingstarted/memcached/
 
 Making security policies on clusters on AWS::
 https://cilium.readthedocs.io/en/v1.5/gettingstarted/aws/
+
+Kubernetes Network Policies in Cilium::
+https://cilium.io/blog/2018/09/19/kubernetes-network-policies/
+
+Layer 7 networking examples::
+https://cilium.readthedocs.io/en/stable/policy/language/#layer-7-examples
```

Comment:
`s/stable/1.5/`?

I would prefer to point only to concrete releases, because we might not follow regularly with upstream updates (well, like now for example :D). Maybe to make it easier we should store Cilium version in some variable and bump it each time we update Cilium?

Reply:
We already have these variables in the attributes file: https://github.com/SUSE/doc-caasp/blob/master/adoc/attributes.adoc

You are talking about this link right? `https://cilium.readthedocs.io/en/v1.5/gettingstarted/aws/`

We could add a "cilium_docs" variable to be able to update this easily?

Reply:
Yes, I'm talking about the link.

What about having variables like these?

```
:cilium_major: 1
:cilium_minor: 5
:cilium_patch: 3
:cilium_version: {cilium_major}.{cilium_minor}.{cilium_patch}
:cilium_shortversion: {cilium_major}.{cilium_minor}
```

And then refering to the `cilium_shortversion` variable in URLs like:

```
https://cilium.readthedocs.io/en/{cilium_shortversion}/policy/language/#layer-7-examples
```